Getting-Started/Setup/Upgrading/general.md

```diff
@@ -60,9 +60,9 @@ There's also the possibility that some files in the /Config folder are new or so
 Up until version 6.0.0 it was necessary to change the version number in ClientDependency.config. This was to clear the cached html/css/js files in the backoffice. Just change the current version number to one that's higher than that. Make sure not to skip this step as you might get strange behaviour in the backoffice otherwise.
 
 ## Merge UI.xml and language files
-Some packages (like Contour) add dialogs to the UI.xml. Make sure to merge those changes back in from your backup during the upgrade so that the packages continue to work. This file can be found in: /Umbraco/Config/Create/UI.xml.
+Some packages (like Umbraco Forms) add dialogs to the UI.xml. Make sure to merge those changes back in from your backup during the upgrade so that the packages continue to work. This file can be found in: /Umbraco/Config/Create/UI.xml.
 
-Packages like Contour and Courier also make changes to the language files located in: /Umbraco/Config/Lang/*.xml (typically en.xml).
+Packages like Umbraco Forms and Courier also make changes to the language files located in: /Umbraco/Config/Lang/*.xml (typically en.xml).
```

Comment:
This is not true of Umbraco Forms as it has it's own language files

Reply:
Thanks a lot @jamescoxhead for the suggestion.
As Contour is another product than Umbraco Forms it's up to Umbraco HQ to accept or to reject this submission.  This might take some time as Codegarden is a source for lots of changes for Umbraco HQ.

With that being said, if you have other things you would like to submit for the documentation don't hesitate to do so!

Reply:
@jamescoxhead - Did you have a chance to double-check here? 😁

Reply:
I can confirm it doesn't happen on the latest version of Forms, but we've got a project with version 6 installed where the language files are updated.